app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -114,6 +114,9 @@ $include-html-global-classes: $include-html-classes;
 // $oil         : #333333;
 // $jet         : #222222;
 // $black       : #000000;
+ $share-purple: #AD1BA1;
+ $share-dark-blue: #033E58;
+ $share-yellow: #FED728;
```

Comment:
Color `#FED728` should be written as `#fed728`<br>Line should be indented 0 spaces, but was indented 1 spaces